src/swarm/neo/client/mixins/ClientCore.d

```diff
@@ -581,10 +581,10 @@ template ClientCore ( )
 
     private RequestId assign ( R, P ) ( P params )
     {
-        static assert(is(P == R.UserSpecifiedParams));
+        static assert(is(P == Const!(R.UserSpecifiedParams)));
```

Comment:
I'd use `:` instead of `==` here, as both mutable and immutable are implicitly convertible to const.